### PR TITLE
fix(utlendingastofnun): small fixes after testing

### PR DESCRIPTION
**Pull request by berglindoma13**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/directorate-of-immigration/citizenship/src/fields/SelectedChildrenExtra/SelectedRepeaterItem.tsx [hunk 1]
```diff
@@ -140,7 +140,7 @@ export const SelectedRepeaterItem: FC<Props & FieldBaseProps> = ({
           setShowMoreQuestions(!showMoreQuestions)
         }}
         disabled={readOnlyFields}
-        defaultValue={repeaterField.hasFullCustody === 'yes' ? YES : ''}
+        defaultValue={repeaterField.hasFullCustody === NO ? NO : ''}
```
Comment (davidkef) [inline libs/application/templates/directorate-of-immigration/citizenship/src/fields/SelectedChildrenExtra/SelectedRepeaterItem.tsx:143]: Could it be that this is suppose to be YES, or was it wrong before

File: libs/application/templates/directorate-of-immigration/citizenship/src/forms/CitizenshipForm/InformationSection/CountriesOfResidenceSubSection.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ import {
 } from '@island.is/application/core'
 import { information } from '../../../lib/messages'
 import { Routes } from '../../../lib/constants'
+import { Answer } from '@island.is/application/types'
```
Comment (davidkef) [inline libs/application/templates/directorate-of-immigration/citizenship/src/forms/CitizenshipForm/InformationSection/CountriesOfResidenceSubSection.ts:8]: unused

File: libs/application/templates/directorate-of-immigration/citizenship/src/lib/dataSchema.ts [hunk 3]
```diff
@@ -107,6 +109,37 @@ export const RemoveableCountrySchema = z
...
+  //   },
+  // )
+  // .refine(
+  //   ({ wasRemoved, dateFrom }) => {
+  //     return wasRemoved === 'true' || (dateFrom && dateFrom.length > 0)
+  //   },
+  //   {
+  //     path: ['dateFrom'],
+  //   },
+  // )
```
Comment (davidkef) [inline libs/application/templates/directorate-of-immigration/citizenship/src/lib/dataSchema.ts:127]: Can't  this be removed

File: libs/application/templates/directorate-of-immigration/citizenship/src/fields/SelectedChildrenExtra/SelectedRepeaterItem.tsx (hunk 1 above)
Comment (berglindoma13) [reply libs/application/templates/directorate-of-immigration/citizenship/src/fields/SelectedChildrenExtra/SelectedRepeaterItem.tsx:143]: yeah it was wrong before

**Review by davidkef — APPROVED** [review]
LGTM

**Pull request merged**